.nycrc.json

```diff
@@ -1,5 +1,19 @@
 {
-  "extends": "@swellaby/nyc-config",
```

Comment:
Seems to have been a lot of issues with nyc configurations of late, and I feel I've lost a lot of time battling weird coverage reports with TypeScript and/or shared configs. 

I'm inlining the shared config values for the time being, but long term we may want to take a look at replacing nyc with c8 (for several reasons)